docs/guide/conditional-validation.md

```diff
@@ -208,10 +208,11 @@ $validator = new Validator(
 );
 ```
 
-### Configuring empty criterias in other rules
+### Configuring empty criteria in other rules
 
-Some rules, like `Required` can't be skipped on empty values - that would violate the whole purpose of it. However, the 
-empty criteria can be configured here too - not for skipping, but for detecting an empty value:
+Some rules, such as `Required` can't be skipped for empty values - that would defeat the purpose of the rule.
```

Comment:
`defeat` seems to be less appropriate here. Some other options - "contradict", "go against".

Reply:
https://idioms.thefreedictionary.com/defeats+the+purpose